cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/AbstractConnectorConversion.java

```diff
@@ -99,7 +99,8 @@ public void convertToDanglingLine(int boundarySide, double r, double x, double g
                 .add()
                 .add();
         }
-        addAliases(dl);
+        dl.addAlias(terminalId(boundarySide), "CGMES.Terminal_Boundary");
+        dl.addAlias(terminalId(boundarySide == 1 ? 2 : 1), "CGMES.Terminal_Network");
         dl.addAlias(boundaryNode, "CGMES." + CgmesNames.TOPOLOGICAL_NODE);
```

Comment:
prefix `CGMES.` should be defined in `Conversion` module and used here and during export, to query the aliases